### DISCUSSION TITLE
How to remove slide animation while switching between the tabs

### DISCUSSION THREAD

**Discussion by sylendra7** [xamarin/XamarinCommunityToolkit#1170] [Q&A]
How to remove slide animation while switching between the tabs in tabview ?

**Comment by Cfun1** [comment]
Have you tried `<xct:TabView IsTabTransitionEnabled="False"` ?
I suppose this should be moved to a [discussion](https://github.com/xamarin/XamarinCommunityToolkit/discussions)
